### PR TITLE
Add contributing document

### PR DESCRIPTION
**Pull request by int00h5525**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Update [CHANGELOG.md](CHANGELOG.md)
+7. Issue that pull request!
+
+## We Deploy to a Variety of Architectures
+MOTH is most often deployed using Docker, but we strive to ensure that it can easily be run outside of a Docker environment. Please ensure that and changes will not break or substantially alter Dockerized deployments and that, conversely, changes will not so substantially tie MOTH to Docker or particular Docker deployment that it becomes impracticle to run MOTH anywhere but inside of Docker
```
Comment (nealey) [inline CONTRIBUTING.md]: impractical

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
```
Comment (nealey) [inline CONTRIBUTING.md]: s/In Short, w/W/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can.
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+## Use a Consistent Coding Style
```
Comment (nealey) [inline CONTRIBUTING.md]: ### Go

Easy: run it through `gofmt`.

File: Dockerfile.moth [hunk 4]
```diff
@@ -7,4 +7,6 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o /mo
 FROM scratch
 COPY --from=builder /mothd /mothd
 COPY theme /theme
+COPY LICENSE.md /LICENSE
```
Comment (nealey) [inline Dockerfile.moth]: Do we need to do this, or would it be better to expose this someplace less obscure (like when you run the binary)? I can honestly say it's never even occured to me to check inside a Docker image for a license, but I can totes see the logic here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (int00h5525) [reply CONTRIBUTING.md]: What's impractical about this?

Comment (nealey) [reply CONTRIBUTING.md]: The way you spelled "impractical".

File: Dockerfile.moth (hunk 4 above)
Comment (int00h5525) [reply Dockerfile.moth]: I don't think that there's a strong reason *not* to, except to save a few K in the Docker image. Where would make more sense to place this file?

**Pull request merged**